docs/components/appContext.js

```diff
@@ -55,8 +53,10 @@ function AppContextProvider({ children }: {| children?: Node |}): Node {
       value={{
         propTableVariant,
         setPropTableVariant,
-        colorScheme,
-        setColorScheme,
+        colorScheme: cookies['gestalt-color-scheme'] || 'light',
```

Comment:
What happens if you change the `gestalt-color-scheme` to something non-existent. E.g. `blue`, does it break completely or are we still rendering the page to the user?